src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,12 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_PID = new Prefix("id/");
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_PRENAME = new Prefix("p/");
+    public static final Prefix PREFIX_FOOD = new Prefix("f/");
+    public static final Prefix PREFIX_FAMILY = new Prefix("c/");
```

Comment:
Similarly, suggest to rename PREFIX_FAMILY to. PREFIX_FAMILY_CONDITION to deter confusion with other family related information and better scalability.

Reply:
Have been updated to `PREFIX_FAMILY_CONDITION`